Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+public enum TransformType {
+    case resetTransforms
+    case transform
+}
+
+public class TransformRecord: NSObject {
+    
+    let transformType : TransformType!
+    
+    public var actionName : String!
+    
+    var useCurrent : Bool! = true
+    
+    let previousValues :  Dictionary<String, Any?>!
+    var currentValues : Dictionary<String, Any?>!
+    
```

Comment:
Why `previousValues` is `let` and `currentValues` is `var`? Will `currentValues` be changed outside of TransformRecord?